testsuite/python/virtual_sites_relative.py

```diff
@@ -382,4 +394,4 @@ def test_zz_pressure_tensor(self):
 
 
 if __name__ == "__main__":
-    ut.main()
+    ut.main(verbosity=2)
```

Comment:
The point of this change is to make this test output more information when it times out. The test is flaky and times out at random quite infrequently in CI (not reproducible outside of CI).